podcast/the-changelog-300.md

```diff
@@ -297,10 +297,7 @@ I feel like in a lot of readmes, a lot of open source website, it'll say right t
 **Adam Stacoviak:** We were pretty close to that one though. I think there were a lot of community members knee deep into the Io and Node fork and bringing the communities back together. From my perspective, I think there were plenty of community members that were what I would consider just peer-to-peer developers, not corporations that were -- I guess they were leading Io, but then they were also bringing it back to this recombination of Io/Node.
 
 **Zed Shaw:** Right, after they got Joyent to sort of agree to their things. Because Joyent wasn't really running the project ethically; they were doing exactly what I was saying.
-
-**Jerod Santo:** \[unintelligible 00:43:19.27\]
-
-**Zed Shaw:** Yeah, they weren't running it well; they were only doing what they wanted and what benefitted them, but then claiming "Oh, this is a community. We're all friends!" So then Io forks for that very reason.
+Yeah, they weren't running it well; they were only doing what they wanted and what benefitted them, but then claiming "Oh, this is a community. We're all friends!" So then Io forks for that very reason.
```

Comment:
This needs a blank line above it and also needs the `**Zed Shaw:**` heading as it comes after my unintelligible statement.